db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL, null=True)
```

Comment:
The `on_delete` parameter for the `race` ForeignKey in the `Skill` model is set to `models.SET_NULL`, which is appropriate if the `Skill` can exist without a `Race`. Ensure this is the intended behavior as per the checklist: 'Use `on_delete=models.CASCADE` if you know that some model doesn't exist without other model.'